README.txt

```diff
@@ -52,8 +52,8 @@ Build Process
 2. Execute the following commands to build Apache Atlas
 
    $ export MAVEN_OPTS="-Xms2g -Xmx2g"
-   $ mvn clean install
-   $ mvn clean package -Pdist
+   $ mvn clean install -DskipTests 
+   $ mvn clean package -Pdist -DskipTests 
```

Comment:
This change does not seem related to the issue with Sqoop.

Reply:
Yeah, I think the unit tests are buggy. The problem with Sqoop is that it is banned in some countries and using proxy is necessary.

Reply:
In that case you need to set your proxy settings. Or disable the tests just locally.

Reply:
You are right! but the readme file in the repository should have detail about tests! Unit tests have problem!

Reply:
The right way is to fix the failing tests, not to update the documentation to ignore them. Or worse - to delete the tests.
If you don't know how to fix the tests then file an issue explaining the problem and leave it to someone who knows what to do. In the meantime use `-DskipTests` **locally**.